pkg/cmd/taskrun/delete.go

```diff
@@ -187,18 +191,23 @@ func deleteTaskRuns(s *cli.Stream, p cli.Params, trNames []string, opts *options
 
 			if !ownerPrFinished(cs, *tr) {
```

Comment:
this can be made like

```
if !ownerPrFinished(cs, *tr) && opts.ForceDelete {
	fmt.Fprintf(s.Out, "warning: Taskrun %s related pipelinerun still running.\n", tr.Name)
}
if !ownerPrFinished(cs, *tr) && !opts.ForceDelete	{
	fmt.Fprintf(s.Out, "TaskRun(s): %s attached to PipelineRun is still running deleting will restart the completed taskrun. Proceed (y/n): ", tr.Name)
	err := opts.TakeInput(s, ""); err != nil {
		continue
	}
}
processedTrNames = append(processedTrNames, tr.Name)
```

Reply:
I think we will still miss condition if ownerPrFinished is true. 
But any will I will optimise and clean it. :+1:

Reply:
no i have moved the append at last, so that case will be handled

Reply:
missed thatone.